sql/_32_damson/cbrd_22783_REGEXP_functions/cases/REGEXP_REPLACE.sql

```diff
@@ -57,26 +57,29 @@ INSERT INTO athlete VALUES
 ,(12163, 'Li Ge', 'M', 'CHN', 'Gymnastics')
 ,(14912, 'Kelly', 'W', 'BRA', 'Football')
 ,(14873, 'Jo Sh', 'M', 'KOR', 'Boxing');
+SELECT * FROM athlete ORDER BY 1;
 
 SELECT 
-REGEXP_REPLACE (name, '[a-d]', '#'), REGEXP_REPLACE (name, '[e-z]', '@') 
-FROM athlete ORDER BY 1 LIMIT 5;
+REGEXP_REPLACE (name, '[a-e]', '#'), REGEXP_REPLACE (name, '[f-z]', '@') 
+FROM athlete ORDER BY 1;
 
-SELECT REGEXP_REPLACE (name, '[a-d]', '#', 6, 0, 'i') from athlete ORDER BY 1 LIMIT 5;
+SELECT REGEXP_REPLACE (name, '[a-e]', '#', 3, 0, 'i') from athlete ORDER BY 1;
+
+SELECT REGEXP_REPLACE (name, '[a-e]', '#', 3, 0, 'c') from athlete ORDER BY 1;
 
 SELECT name FROM athlete 
 WHERE LENGTH (REGEXP_REPLACE (name, '\s', '')) < 5 ORDER BY 1;
 
 WITH V_TEST AS (SELECT 'hello@cubrid.com' EMAIL)
 SELECT REGEXP_REPLACE(EMAIL, 'hello', 'cub') AS "id" FROM V_TEST ORDER BY 1;
 
-CREATE TABLE new_athlete ( encrypted_name VARCHAR ) AS SELECT REGEXP_REPLACE (name, '[a|e|i|o|u]', '#') AS encrypted_name from athlete ORDER BY 1 LIMIT 10;
+CREATE TABLE new_athlete ( encrypted_name VARCHAR ) AS SELECT REGEXP_REPLACE (name, '[a|e|i|o|u]', '#') AS encrypted_name from athlete ORDER BY 1;
 SELECT * from new_athlete ORDER BY 1;
 DROP TABLE new_athlete;
 
 CREATE TABLE new_athlete (encrypted_name VARCHAR);
-INSERT INTO new_athlete SELECT REGEXP_REPLACE (name, '[a|e|i|o|u]', '#') AS encrypted_name from athlete ORDER BY 1 LIMIT 10;
+INSERT INTO new_athlete SELECT REGEXP_REPLACE (name, '[a|e|i|o|u]', '#') AS encrypted_name from athlete ORDER BY 1;
 SELECT * from new_athlete ORDER BY 1;
-DROP TABLE new_athlete;
 
+DROP TABLE new_athlete;
```

Comment:
`DROP TABLE` may be unstable If a table is not created by an error. please use `DROP TABLE if exists` instead of `DROP TABLE`